pkg/commands/git_commands/sync.go

```diff
@@ -49,10 +49,14 @@ func (self *SyncCommands) Push(task gocui.Task, opts PushOpts) error {
 	return cmdObj.Run()
 }
 
+func (self *SyncCommands) fetchCommandBuilder(fetchAll bool) *GitCommandBuilder {
+	return NewGitCmd("fetch").
+		ArgIf(fetchAll, "--all").
+		ArgIf(!self.version.IsOlderThan(2, 29, 0), "--no-write-fetch-head")
```

Comment:
Thoughts on introducing `self.version.AtLeast(2, 29, 0)`?

Reply:
Good idea, I added a separate commit (a6c908bc8366) that adds it and uses it in a few other places.